src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/TransactionService.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.companieshouse.overseasentitiesapi.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+import uk.gov.companieshouse.overseasentitiesapi.client.ApiClientService;
+import uk.gov.companieshouse.overseasentitiesapi.exception.ServiceException;
+import java.io.IOException;
+
+@Service
+public class TransactionService {
+
+    private final ApiClientService apiClientService;
+
+    @Autowired
+    public TransactionService(ApiClientService apiClientService) {
+        this.apiClientService = apiClientService;
+    }
+
+    public Transaction getTransaction(String transactionId, String passthroughHeader) throws ServiceException {
```

Comment:
Presume call to create a transaction will follow later?

Reply:
creating will be done outside of our api, but we will need to 'lift and shift' the update transaction method in the future

Reply:
Oh yes, that's true. All good.